pandas/io/formats/latex.py

```diff
@@ -10,56 +11,87 @@
 from pandas.io.formats.format import DataFrameFormatter, TableFormatter
 
 
-class LatexFormatter(TableFormatter):
-    """
-    Used to render a DataFrame to a LaTeX tabular/longtable environment output.
+class RowStringConverter:
+    r"""Converter for dataframe rows into LaTeX strings.
 
     Parameters
     ----------
     formatter : `DataFrameFormatter`
-    column_format : str, default None
-        The columns format as specified in `LaTeX table format
-        <https://en.wikibooks.org/wiki/LaTeX/Tables>`__ e.g 'rcl' for 3 columns
-    longtable : boolean, default False
-        Use a longtable environment instead of tabular.
+        Instance of `DataFrameFormatter`.
+    multicolumn: bool, optional
+        Whether to use \multicolumn macro.
+    multicolumn_format: str, optional
+        Multicolumn format.
+    multirow: bool, optional
+        Whether to use \multirow macro.
 
-    See Also
-    --------
-    HTMLFormatter
     """
 
     def __init__(
-        self,
-        formatter: DataFrameFormatter,
-        column_format: Optional[str] = None,
-        longtable: bool = False,
-        multicolumn: bool = False,
-        multicolumn_format: Optional[str] = None,
-        multirow: bool = False,
-        caption: Optional[str] = None,
-        label: Optional[str] = None,
-        position: Optional[str] = None,
+        self, formatter, multicolumn=False, multicolumn_format=None, multirow=False,
```

Comment:
looking across to `LatexFormatter` I think we want type hints here

Reply:
Added type hints.